functions/x-header-parameter.js

```diff
@@ -0,0 +1,10 @@
+module.exports = (targetVal, opts) => {
+    headers = targetVal.parameters.filter(param => (param.name == "X-Caller-Name" || param.name == "X-Event-ID"));
```

Comment:
I guess we need `&&` instead of `||` as the error message already says `and`.

Reply:
i check if the result headers has the size 2. It's not a prefect test, because 2 times X-Caller-Name would be good as well.